spaceship.zsh

```diff
@@ -525,6 +533,30 @@ spaceship_hg() {
     "$SPACESHIP_HG_SUFFIX"
 }
 
+# EMBER
+# Show current version of ember, exception system.
+spaceship_ember() {
+  [[ $SPACESHIP_EMBER_SHOW == false ]] && return
+
+  # Show EMBER status only for folders w/ ember-cli-build.js files
+  [[ -f ember-cli-build.js ]] || return
+
+  local ember_version
+
+  if _exists ember version; then
```

Comment:
It could be simplified as:

```zsh
_exists ember || return

local ember_version=$(ember version | grep -Po "(?<=ember-cli\: ).*$")
[[ $ember_version == "system" || $ember_version == "ember" ]] && return
```